src/t2iapi/device/types.proto

```diff
@@ -50,6 +50,15 @@ enum MdsOperatingMode{
     MDS_OPERATING_MODE_MAINTENANCE = 3;
 }
 
+/*
+Represents a type of Descriptor.
```

Comment:
unnecessary capitalization _descriptor_, same for the comment in DescriptorType

Reply:
Corrected.